### ISSUE TITLE
在微信小程序端训练时出错

### ISSUE THREAD

**Issue by zju-zry** [tensorflow/tfjs-wechat#106]
To get help from the community, we encourage using Stack Overflow and the tensorflow.js tag.

TensorFlow.js version 3.7
tfjs-wechat plugin version 0.2.0

以下是我打算在小程序端测试使用的代码：

`
import * as tf from '@tensorflow/tfjs-core'
// import * as tfb from '@tensorflow/tfjs-backend-webgl'

export default() => {

    const a = tf.variable(tf.scalar(Math.random()));
    const b = tf.variable(tf.scalar(Math.random()));
    const c = tf.variable(tf.scalar(Math.random()));
    const d = tf.variable(tf.scalar(Math.random()));
 

    function predict(x) {
        // y = a * x ^ 3 + b * x ^ 2 + c * x + d
        return tf.tidy(() => {
            let y = tf.add(tf.mul(a,tf.pow(x,tf.scalar(3))) // a * x^3
                ,(tf.mul(b,tf.square(x))) // + b * x ^ 2
                ,(tf.mul(c,x)) // + c * x
                ,(d)); // + d
            x.print()
            y.print()
            return y
        });  
    }

    function loss(predictions, labels) {
        // Subtract our labels (actual values) from predictions, square the results, and
        // take the mean.
      const meanSquareError = tf.mean(tf.square(tf.sub(predictions, labels)));
      return meanSquareError;
    }

    function train(xs, ys, numIterations = 75) {

        const learningRate = 0.5;
        const optimizer = tf
            .train
            .sgd(learningRate);

        for (let iter = 0; iter < numIterations; iter++) {
            optimizer.minimize(() => {
                const predsYs = predict(xs);
                return loss(predsYs, ys);
            });
        }
    }

    function generateData(numPoints, coeff, sigma = 0.04) {
        const gety = x =>  coeff.a * x * x * x + coeff.b * x * x + coeff.c * x + coeff.d
        let ans = {xs:[],ys:[]}
        for (let i=0;i<numPoints;i++){
            let x = Math.random()
            let y = gety(x)
            ans.xs.push(x)
            ans.ys.push(y)
        }
        ans.xs =tf.tensor1d(ans.xs)
        ans.ys =tf.tensor1d(ans.ys)
        return ans
    }

    const trueCoefficients = {a: -.8, b: -.2, c: .9, d: .5};
    const trainingData = generateData(100, trueCoefficients);

    train(trainingData.xs, trainingData.ys, 70);

}
`

**Comment by zju-zry** [comment]
遇到的问题是
`

VM82 WAService.js:2 Error: Cannot compute gradient: gradient function not found for Mean.
    at backpropagateGradients (vendors.js:33063)
    at vendors.js:3871
    at vendors.js:3336
    at Engine.scopedRun (vendors.js:3346)
    at Engine.tidy (vendors.js:3335)
    at Engine.gradients (vendors.js:3867)
    at variableGrads (vendors.js:5019)
    at SGDOptimizer.computeGradients (vendors.js:31975)
    at SGDOptimizer.minimize (vendors.js:31931)
    at train (._src_pages_task_schedule_fit_index.js:40)(env: macOS,mp,1.05.2105170; lib: 2.18.0)


`

**Comment by xuyixun21** [comment]
hello,do you slove  this problem?
